plugins/pencilblue/controllers/index.js

```diff
@@ -280,7 +280,7 @@ module.exports = function IndexModule(pb) {
         ats.registerLocal('author_position', content.author_position ? content.author_position : '');
         ats.registerLocal('media_body_style', content.media_body_style ? content.media_body_style : '');
         ats.registerLocal('comments', function(flag, cb) {
-            if (isPage || !contentSettings.allow_comments || !content.allow_comments === false) {
+            if (isPage || !contentSettings.allow_comments || !content.allow_comments) {
```

Comment:
Does this also need to be changed/updated in Portfolio, Jennifer, and DevBlog themes?